addons/sourcemod/scripting/include/ZR_RepeatKillDetector.inc

```diff
@@ -0,0 +1,36 @@
+#if defined _zr_repeatkilldetector_included
+    #endinput
+#endif
+#define _zr_repeatkilldetector_included
+
+// Version information
+#define ZR_RKD_V_MAJOR   1
+#define ZR_RKD_V_MINOR   2
+#define ZR_RKD_V_PATCH   0
+
+#define ZR_RKD_VERSION   "1.2.0"
```

Comment:
i dont know if its possible but could you try something like this:
```
#define ZR_RKD_VERSION ZR_RKD_VERSION_MAJOR ... "." ... ZR_RKD_VERSION_MINOR ... "." ... ZR_RKD_VERSION_PATCH
```

Reply:
Good point, done.